packages/rest-typings/src/v1/users.ts

```diff
@@ -1,26 +1,140 @@
 import type { ITeam, IUser } from '@rocket.chat/core-typings';
+import Ajv from 'ajv';
+
+const ajv = new Ajv({
```

Comment:
Looking at the times we're using this, would be good to have it centralized, in case we ever want to add/change a config option 🙈 

(Not sure if on the scope of this PR, but)

Reply:
yep I'm planning to centralize them